### PR TITLE
Upload page added to Upload/Delete storage files

### PR DESCRIPTION
**Pull request by marybozorgi**
Upload page added to Upload/Delete storage files in bucket "info" 
policies added to Storage bucket "info" to allow "Insert" and "Delete" in addition to "Select"

### CODE REVIEW

**Review by jnakaso — CHANGES_REQUESTED** [review]
Looks good.  Just a couple of nits:
1. console.log needs to be removed.
2. Can "any" be replaced with concrete types?

File: src/pages/UploadPage.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const File = (id: string, name: string, size: number, mimetype: string) => 
+  { return { name: name, id: id, metadata:{size: size, mimetype: mimetype}, created_at: moment().toDate().toISOString() } }
+
+
+  const handleUpload = (event: any) => {
+
+    const file = event.target.files[0];
+    storageService.uploadFile(file)
+      .then(resp => {
+        console.log(resp);
```
[the remark this answers is not in the document]
Comment (marybozorgi) [reply src/pages/UploadPage.tsx]: sorry for late changes,
I just today noticed that there is change request
did not get any notification for it 
thanks

File: src/services/storageService.ts [hunk 2]
```diff
@@ -24,6 +33,45 @@ class StorageService {
...
+            .remove([fileName])
+            .then(resp => {
+                if (resp.error) {
+                    throw new Error(resp.error.message)
+                }
+                return resp.data
+            })
+    }
+
+    uploadFile = async (file: any): Promise<any> => {
```
Comment (jnakaso) [inline src/services/storageService.ts:62]: This probably should be typed as File

**Review by jnakaso — APPROVED** [review]
Looks okay.

File: src/services/storageService.ts (hunk 2 above)
Comment (marybozorgi) [reply src/services/storageService.ts:62]: the file object format while uploading is different from file object stored in storage
so I could not use same File type for it

**Pull request merged**